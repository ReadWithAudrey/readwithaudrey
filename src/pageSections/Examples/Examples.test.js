import React from 'react'
import renderer from 'react-test-renderer'
import Examples from './Examples'

describe('Examples component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<Examples />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
