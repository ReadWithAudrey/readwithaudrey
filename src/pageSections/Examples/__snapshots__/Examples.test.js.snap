// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Examples component snapshot test renders correctly 1`] = `
<div
  className="sc-bdVaJa bg-white flex flex-column items-center justify-between ph4 page-padding montserrat w-100 fePDys"
>
  <h1
    className="f2 pink tc"
  >
    Recent reading partners
  </h1>
  <ul
    className="tl"
  >
    <li
      className="pv2"
    >
      A musician and a school teacher read Musicophilia, by Oliver Sacks.
    </li>
    <li
      className="pv2"
    >
      A counsellor and medical doctor read Attachment Theory by J Bowlby.
    </li>
    <li
      className="pv2"
    >
      A retired judge and a history student read Darkest Hour, by Anthony McCarten.
    </li>
    <li
      className="pv2"
    >
      A young playwright and an artist read short stories by Hector Munro.
    </li>
    <li
      className="pv2"
    >
      A photographer and a retired nurse read Language of Kindness by Christie Watson.
    </li>
    <li
      className="pv2"
    >
      A film maker and a furniture restorer read The Wind in My Hair by Masih Alinejad.
    </li>
  </ul>
  <p
    className="f4 tc"
  >
    We take great care to try to identify the right reading partner for you.
  </p>
<<<<<<< HEAD
  <Link
    className="no-underline white w-100"
    to="/Form1/"
  >
    <button
      className="sc-htoDjs bg-pink hover-bg-orange white pa2 br2 w-50 h3 f3 mt4 center fHOsVa"
    >
      Get started
    </button>
  </Link>
=======
>>>>>>> master
</div>
`;
